@xen-orchestra/lite/src/components/form/FormByteSize.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <FormInputGroup>
+    <FormInput v-model="size" type="number" />
+    <FormSelect v-model="prefix">
+      <option
+        v-for="currentPrefix in availablePrefixes"
+        :key="currentPrefix"
+        :value="currentPrefix"
+      >
+        {{ currentPrefix }}B
+      </option>
+    </FormSelect>
+  </FormInputGroup>
+</template>
+
+<script lang="ts" setup>
+import FormInput from "@/components/form/FormInput.vue";
+import FormInputGroup from "@/components/form/FormInputGroup.vue";
+import FormSelect from "@/components/form/FormSelect.vue";
+import humanFormat, { type BinaryPrefix } from "human-format";
+import { ref, watch } from "vue";
+
+const props = defineProps<{
+  modelValue: number;
+}>();
+
+const emit = defineEmits<{
+  (event: "update:modelValue", value: number): number;
+}>();
+
+const { value: initialSize, prefix: initialPrefix } = humanFormat.raw(
+  props.modelValue,
+  { scale: "binary" }
+);
+
+const size = ref<number>(initialSize);
+const prefix = ref<BinaryPrefix>(initialPrefix);
+const availablePrefixes: BinaryPrefix[] = [
+  "",
+  "Ki",
+  "Mi",
+  "Gi",
+  "Ti",
+  "Pi",
+  "Ei",
+  "Zi",
+  "Yi",
```

Comment:
Do you think we should have all these choices?

Reply:
Good question 🤔 

Maybe we should add optional props for min/max prefix?

Something like `<FormByteSize min-prefix="Mi" max-prefix="Gi" />`

What do you think?

ping @MathieuRA @pdonias

Reply:
In XO 5, we have K, M and G, and we've never had any requests to change it so I guess we can use that at least by default. Or maybe T would also be necessary for big data SRs?

Reply:
So let's start with `Ki`, `Mi` and `Gi` for now and we'll still be in time to add more later if requested.